bin/compile

```diff
@@ -127,7 +127,7 @@ format_cnb_output() {
 
 # Run CNB bin/build with formatted output
 run_build() {
-	"${CNB_BUILDPACK_DIR}/bin/build" "${CNB_BUILD_DIR}/layers" "${CNB_BUILD_DIR}/platform" "${CNB_BUILD_DIR}/buildpack_plan.toml" | format_cnb_output
+	"${CNB_BUILDPACK_DIR}/bin/build" "${CNB_BUILD_DIR}/layers" "${CNB_BUILD_DIR}/platform" "${CNB_BUILD_DIR}/buildpack_plan.toml" 2>&1 | format_cnb_output
```

Comment:
As an alternative, Bash 4+ supports `|&` which is a shorthand for `2>&1 |`:
https://www.gnu.org/software/bash/manual/html_node/Pipelines.html

e.g.:
https://github.com/heroku/heroku-buildpack-python/blob/b90a4755ae2008a5cdae2b32632a9fb4da49db0d/lib/pip.sh#L116